WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
Okay. I'm adding back cultural and religious requirements to "K_CAUCASIAN_ALBANIA" now.